client/boot/dev-modules.js

```diff
@@ -1,26 +1,13 @@
 /**
- * External dependencies
- *
  * @format
  */
 
-import ReactClass from 'react/lib/ReactClass';
-
 /**
  * Internal dependencies
  */
 import config from 'config';
-import RenderVisualizerMixin from 'lib/mixins/render-visualizer';
 
 export default function() {
-	if ( config.isEnabled( 'render-visualizer' ) ) {
```

Comment:
I’m on mobile so it’s hard to verify for, but did you check if ‘render-visualizer’ flag needs to be removed from config files?

Reply:
Yup! No instances of `render-visualizer` were present in the config files.